Production/crab/configs/2016UL/tau_cp/Higgs.txt

```diff
@@ -0,0 +1,7 @@
+sampleType=MC_UL16 selector selector=genTauTau
+
+GluGluHToTauTau_M125   /GluGluHToTauTau_M125_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v2/MINIAODSIM
```

Comment:
for 2016 there are two periods: preVFP and postVFP. Currently configs for preVFP samples are missing. E.g.
```
/GluGluHToTauTau_M125_TuneCP5_13TeV-powheg-pythia8/RunIISummer20UL16MiniAODAPVv2-106X_mcRun2_asymptotic_preVFP_v11-v2/MINIAODSIM
```